src/app/View/Components/common/Base.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\View\Components\common;
+
+use Closure;
+use Illuminate\Contracts\View\View;
+use Illuminate\View\Component;
+
+class Base extends Component
+{
+    public string $title;
+
+    /**
+     * <body>までのHTMLを生成するコンポーネント
+     *
+     * @param string $title タイトル
+     */
+    public function __construct(string $title = null)
+    {
+        if (is_null($title)) {
+            $this->title = '';
+            return;
+        }
+        $this->title = ' - '. $title;
```

Comment:
【IMO】
コンストラクターで return を使用することはおすすめしません。
なので、これで三項演算子を使った方が良いと思います。
```suggestion
        $this->title = is_null($title) ? '' : ' - '. $title;
```